pkg/kubectl/cmd/util/clientcache.go

```diff
@@ -125,3 +131,71 @@ func (c *ClientCache) ClientForVersion(version *unversioned.GroupVersion) (*clie
 
 	return kubeclient, nil
 }
+
+func (c *ClientCache) KubeClientSetForVersion(version *unversioned.GroupVersion) (kube_client_1_3.Interface, error) {
```

Comment:
It was intent to be used in describers which may need both the kube client and the fed client.
Since we will have only clusters now, I will remove it now.